assets/scss/custom.scss

```
@@ -512,3 +512,24 @@ dd {
     border-radius: 0.25rem;  // was $border-radius, but that var isn't accessible here.
   }
 }
+
+/* Styling for the summary-table shortcode */
+.summary-table {
```

I think this makes the `!important` on the `padding` unnecessary:

```suggestion
table.summary-table {
```